client/account/go.mod

```diff
@@ -0,0 +1,10 @@
+module github.com/decred/dcrdex/client/account
+
+go 1.13
+
+replace github.com/decred/dcrdex/server/account => ../../server/account
+
+require (
+	github.com/decred/dcrd/dcrec/secp256k1 v1.0.3
+	github.com/decred/dcrdex/server/account v0.0.0-00010101000000-000000000000
```

Comment:
Similar mod cleanup with client/account too, but it's not terribly important at this stage.

```diff
+github.com/decred/dcrdex/server/account v0.0.0-20191014151634-318621ff1b98
-github.com/decred/dcrdex/server/account v0.0.0-00010101000000-000000000000
```

and a tidy for the sum while at it. Thanks!